pages/about/translating.md

```diff
@@ -48,68 +50,92 @@ To get announcements related to WAI translations, subscribe to the WAI Translati
+## Ways to contribute
 
-## Translation Instructions
+You can contribute to WAI translations in many ways:
 
-**Scope:**
-* These instructions cover web pages with a URI that begins with www.w3.org/WAI
-* For web pages that begin with www.w3.org/TR/ or something else, there is a different process that is introduced in [TR & Authorized W3C Translations below](#tr).
+[Providing video captions/subtitles](/about/translating/resources/subtitles/)
+: This is the easiest way to start contributing, especially if you are not technical.
 
-**Translator background** &mdash; we prefer translators to be:
-* native speakers
-* familiar with accessibility terminology in their language
-* comfortable editing a file with code
+[Translating a WAI resource](/about/translating/resources/)
+: You are welcome to provide a translation for educational resources on WAI website (web pages with a URL that begins with www.w3.org/WAI). It is best if you are comfortable editing a file with code.
 
-To avoid overlapping work:
-* please do **not** translate files from the web
-* follow the [step-by-step guide](/about/translating/resources/) to _get the right file to translate_ and to ensure that the resource is ready for you to translate.
+[Reviewing WAI resources translations](/about/translating/reviewing/)
+: You can help spot issues in translations before publication, even if you are not technical or do not know the translated language.
 
-### If you want to translate a WAI resource:
+[Creating a WCAG Unofficial translation {% include_cached external.html label="Different page format" %}](https://www.w3.org/Consortium/Translation/)
+: You can host an unofficial translation of the Web Content Accessibility Guidelines (WCAG) after following a light process. Unofficial translations are not hosted on W3C website nor endorsed by W3C.
 
-- Follow instructions in [[Step-by-Step Guide to Translating WAI Resources]](/about/translating/resources/).
+[Creating a WCAG Authorized translation](/about/translating/wcag/)
+: You can develop an Authorized translation of WCAG that will be published on W3C website. It involves a formal process, coordination with stakeholder organizations, and a thorough review.
 
-### If you want to volunteer to review a translation:
+We encourage you to keep up on translations work by [subscribing to the WAI Translations mailing list](mailto:public-wai-translations-request@w3.org?subject=subscribe).
 
-- Follow instructions in [[Reviewing a Translation]](/about/translating/reviewing/).
+## Process
 
-We encourage you to keep up on related translations work by [subscribing to the WAI Translations mailing list](mailto:public-wai-translations-request@w3.org?subject=subscribe).
-   
-### Important notes
+All contributions must go through the following main steps before publication. These steps are described in more detail in each policy, and additional steps may be necessary for some documents.
```

Comment:
I think 'policy' is a bit too formal. Does it work to replace 'policy' with 'section' or 'process' or something else?